modules/ROOT/pages/enable-and-disable-a-public-portal.adoc

```diff
@@ -0,0 +1,68 @@
+= Enable and Disable a Public Portal
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All public portals in Exchange created after July 2022 are disabled by default. You cannot sign in to view assets or share new assets on your public portal unless you enable them. You must be an Exchange administrator to enable or disable a public portal.
```

Comment:
```suggestion
All public portals in Exchange are enabled by default. If disable your public portal, you cannot sign in to view assets or share new assets on your public portal unless you re-enable them. You must be an Exchange administrator to enable or disable a public portal.
```

Reply:
After a QUAD meeting this was decided:

Reply:
Should we plan for Phase II of "Ability to disable public portal"?
Jul 12 Quad meeting: We'll plan Phase II for Q3 + keep public portals enabled by default until Phase II is completed

Reply:
So we should change this text

Reply:
Thanks for letting me know. I will update the content.

Reply:
updated